src/test/java/com/devonfw/sample/archunit/DevonPackage/PackageStructure.java

```diff
@@ -0,0 +1,453 @@
+package com.devonfw.sample.archunit.DevonPackage;
```

Comment:
In Java packages are always strictly written in lower_case_only.
So `DevonPackage` is exepcted to be the name of a class/type but not a package segment.

Reply:
Just refactored it a few seconds ago.